src/intervals/arithmetic.jl

```diff
@@ -155,6 +155,25 @@ end
 
 //(a::Interval, b::Interval) = a / b    # to deal with rationals
 
+if VERSION >= v"0.6.0-dev"
```

Comment:
You mean `isdefined(:Iterators)`? I don't know how to find the exact cutoff where this change was made. In any case, if it's a dev version isn't the expectation that people will keep updating it to the latest version?